(Update) Hidden/Invis Characters (#10424)
harness/developer-hub: docs/continuous-integration/use-ci/run-step-settings.md

If your pipeline runs tests, you can [view test reports](./run-tests/viewing-tests.md) on the Build details page.

## Hidden/Invisible Characters
An issue customers may experience are unexplained phenomenon within their scripts due to hidden or invisible characters. These characters often appear when pasting from non-plain text resources, and can cause errors in how the script operates.   Harness has a function to display invisible characters which is enabled by default.

> End users should see a highlighted space within their scripts, if an invisible character is contained:

Reviewer: ```suggestion
When invisible characters are present, users will see a highlighted space in their scripts.
```

![](./static/invisiblechr-01.png)

> They can then manage and see what character is contained by hovering over the highlight and selecting to `adjust settings`

Reviewer: ```suggestion
They can hover over the highlight to view the character and click "Adjust settings" to manage the display.
```
